tests/orion/schemas/test_core.py

```diff
@@ -95,6 +95,14 @@ async def test_multiple_message_template_variables_are_validated(self):
 
 
 class TestFlowRunPolicy:
+    class OldFlowRunPolicy(PrefectBaseModel):
```

Comment:
I love this idea of including the previous version too, so cool

Reply:
Should this go farther and identify in the name which version of the model this is?

Reply:
Ah, I see it's in the test docstring, maybe we can change the name of the class if we ever need to do so (i.e. there are _two_ old flow run policies we test...)